openfeature-provider-local/src/main/java/com/spotify/confidence/flags/resolver/AccountResolver.java

```diff
@@ -48,7 +47,6 @@ public class AccountResolver {
       new ResolveFlagListener() {};
 
   public AccountResolver(
-      MaterializationStore materializationStore,
```

Comment:
I was thinking to just pass a no-op materialization store. It might come in handy later and I think in principle we should avoid changing any copied code.